catalog/task/validate-single-component/0.1/validate-single-component.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: validate-single-component
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task validates that the snapshot only contains a 
+    single component, The pipeline should fail otherwise.
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+      type: string
+      default: "snapshot_spec.json"
```

Comment:
This doesn't align with the README. I am fine with the default, but if there is a default, the parameter is optional

Reply:
Ah, I got it. Thanks for pointing it out.